---
layout: post
title:  compareMS2, Take Two
categories: [mass spectrometry, proteomics]
excerpt: In a recently published JPR paper, we describe how sets of tandem mass spectra can be compared to each other, without any prior knowledge about genomes, protein sequences or spectra. So what, you ask?
---
In a recently published [JPR paper](https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00457), we describe how sets of tandem mass spectra can be compared to each other, without any prior knowledge about genomes, protein sequences or spectra. So what, you ask? While it is true that most model systems have well annotated genomes and reference proteomes, and sequencing a single organism is getting cheaper and faster all the time, there are still millions of species waiting to be sequences. If studying many species, this would still be a significant cost. Comparing spectra directly, direclty provides information on data quality, the evolutionary relationship between species, and help set suitable parameters for searching the tandem mass spectra against a sequence database (or a spectral library predicted from one). compareMS2 2.0 is free, open source, and as user friendly as we managed to make it (though it can certainly be improved in this regard). It usually produces reasonable results with all parameters at their default.

compareMS2 is avaiable on GitHub: [https://github.com/524D/compareMS2](https://github.com/524D/compareMS2).
&nbsp;  
&nbsp;  
&nbsp; 
